Port to GNOME 45 (breaking change), remove obsolete code

msabur opened the pull request:
Ported the extension to GNOME 45. Due to the changes in GNOME 45, this means the code **will no longer work on older GNOME versions**. I suggest keeping the old code in another branch to allow installation on older GNOME versions.

Tested with GNOME 45.0 (Wayland).

Here's info about how extensions changed in GNOME 45: https://gjs.guide/extensions/upgrading/gnome-shell-45.html

Jazak Allah khairan for your work on this extension!

msabur committed:
breaking: Port to Gnome 45, remove unused code

msabur committed:
Update changelog

msabur committed:
use timeout_add_seconds instead of setTimeout

AbdelrahmanElhosiny commented:
jazak allah kol khair for this port, I was looking for this for days.
working fine on Ubuntu 23.10 (xorg)

raihan2000 commented:
@msabur thanks for the porting stuff also for the suggestions to keep old code in another branch, It will go in master branch to keep uptodate with gnome version

raihan2000 committed:
Add CLI installation steps

this commits add complete installation steps for new and old gnome version from CLI

raihan2000 commented:
Sorry for late merging this. Thank you for the port to 45
